src/style.css

```diff
@@ -1,3 +1,154 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.card {
```

Comment:
<img width="508" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/62164f67-4efd-481a-a864-f958fdef53e6">

Reply:
<img width="768" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/299970cd-9d17-4666-a4f7-c0b35a6ec023">
not fixed